post-deployment/openstack/monitoring-sa.yml

```diff
@@ -0,0 +1,47 @@
+---
+- hosts: localhost
+  tasks:
+  - name: Create monitoring service account
+    k8s:
+      state: present
+      definition:
+        apiVersion: v1
+        kind: ServiceAccount
+        metadata:
+          name: monitoring
+          namespace: openshift-monitoring
+
+  - name: Add cluster-reader permissions to monitoring sa
+    command: oc adm policy add-cluster-role-to-user cluster-reader -z monitoring -n openshift-monitoring
+
+  - name: Register monitor var
+    k8s_info:
+      api_version: v1
+      kind: ServiceAccount
+      name: monitoring
+      namespace: openshift-monitoring
+    register: monitor_sa
```

Comment:
Could you register this when the service account is created? It looks like the k8s module returns some information about objects it creates.

Reply:
I did attempt it but you can only register the details that contain the secrets using the k8s_info module, the k8s module on create doesn't return the necessary values.

Reply:
Fair enough :)